featuretools/variable_types/variable.py

```diff
@@ -257,6 +257,24 @@ class EmailAddress(Variable):
     _default_pandas_dtype = str
 
 
+class URL(Variable):
+    """Represents a valid web url"""
+    _dtype_repr = "url"
+    _default_pandas_dtype = str
+
+
+class PhoneNumber(Variable):
+    """Represents a valid phone number"""
```

Comment:
We should put more information in this docstring. Saying that having a phone number with/without area and country code is okay, as well as a phone number with parentheses.